bin/diagnostic.js

```diff
@@ -1 +1,15 @@
 'use strict';
+const ask = require('../lib/ask.js');
+
+let answer = 'not empty';
+
+while (answer !== '') {
+  answer = ask("Guess my secret? ");
+  if (answer === 1 >= 10) {
```

Comment:
This will be interpreted as ```answer === (1 >= 10)```, which evaluates to ```answer === false```, which results in a strict comparison between a string and a boolean.